pkg/config/parameters.go

```diff
@@ -704,6 +704,11 @@ type Parameters struct {
 	// from k8s endpoint slices. defaults to true,
 	// if false then reading endpoint data from the k8s endpoints.
 	FeatureFlags []string `yaml:"featureFlags,omitempty"`
+
+	// OMEnforcedHealthListener holds configuration for an envoy listener
+	// that enforces the overload manager actions, like global downstream
+	// connection limits.
```

Comment:
nit: add a note that the address configured here must differ from the metrics and health listeners if this health config is enabled